### DISCUSSION TITLE
Mobile Friendly

### DISCUSSION THREAD

**Discussion by khoogeveen60538** [microweber/microweber#973] [General]
I am creating a site via a Linode Server and my site is not showing mobile friendly on phones or tablets.  Is there a setting that needs to be changed?

**Comment by phausleitner** [comment]
I think that even all free themes are mobile-optimised. Once I accidentally hid the mobile menu in CSS.. haven't you made such changes or so?

**Reply by khoogeveen60538** [reply]
No changes.  Have not even logged into linux, just used the web interface to set up the website

**Reply by phausleitner** [reply]
It shouldn't be a server problem.. If possible, maybe link to a live site would be helpful
